build-tools/packages/build-cli/src/BasePackageCommand.ts

```diff
@@ -34,8 +35,18 @@ export abstract class PackageCommand<
 		...BaseCommand.flags,
 	};
 
-	private filterOptions: PackageFilterOptions | undefined;
-	private selectionOptions: PackageSelectionCriteria | undefined;
+	protected filterOptions: PackageFilterOptions | undefined;
+	protected selectionOptions: PackageSelectionCriteria | undefined;
+
+	/**
+	 * An array of packages selected based on the selection criteria.
+	 */
```

Comment:
Thoughts?
```suggestion
	/**
	 * An array of packages selected based on the selection criteria.
	 *
	 * @remarks
	 * These are not necessarily the packages that get acted on.
	 * Filter criteria are first applied to them to determine that.
	 */
```

Reply:
Good idea. I'll add these details, just want to tweak the wording.